src/api/common_api.js

```diff
@@ -310,7 +310,7 @@ module.exports = {
                     type: 'array',
                     items: {
                         type: 'object',
-                        required: ['Effect', 'Action', 'Principal', 'Resource'],
+                        required: ['Effect'],
```

Comment:
I understand it is a simple way to solve it, but I'm not sure if we won't "pay" for it by validating the policies in the bucket server when we could do it before.